helm/reana/templates/reana-server.yaml

```diff
@@ -54,9 +54,10 @@ spec:
           - mountPath: {{ .Values.shared_storage.shared_volume_mount_path }}
             name: reana-shared-volume
           {{- range $workspace_paths := .Values.workspaces.paths }}
-          {{- if not (eq $workspace_paths "/var/reana")}}
-          - name: {{ $workspace_paths | quote |replace "/" "" }}
-            mountPath: {{ $workspace_paths }}
+          {{- if not (eq $workspace_paths "/var/reana:/var/reana")}}
```

Comment:
Minor: I was checking the deployment without this if statement and it worked without any problems for me. Is this statement here because of the problems you mentioned when mounting "/var/reana" twice?